openshift/machine-config-operator
# Render baremetal bootstrap configs

**- What I did**
This moves the baremetal infra bootstrap static pod configurations to MCO, which is necessary so the installer can look up the images from the release and pass them in. Previously the pod definitions were in the installer itself, which meant they were written before bootkube.sh was run and could not use the standard method to retrieve the images. This also moves all of the baremetal infra templates into the same repo and converts them to use the MCO templating just like the master versions.

**- How to verify it**
Remove the existing baremetal systemd services from the installer and deploy with this series of commits. Coredns and keepalived should be deployed on the bootstrap node as static pods.

**- Description for the changelog**
Baremetal infra static pod definitions are now deployed during the machine-config-operator bootstrap phase.
